Code/fbuild.bff

```diff
@@ -38,7 +38,7 @@ Compiler( 'Compiler-x64Clang' )
 }
 Compiler( 'Compiler-x64-Linux' )
 {
-    .Executable     = '/usr/bin/gcc'
+    .Executable     = '/usr/bin/g++'
```

Comment:
This change will result in .c files being compiled as C++. In case of LZ4/xxHash (which are bundled with FASTBuild) this doesn't lead to new errors or warnings, but it might change code generation (for better or worse).

Reply:
Is there a way to define 2 different compilers, one for C and the other one for C++ code?

Reply:
It is possible to define two compilers by having two Compiler() sections in the bff. However, a simpler way is to keep the same compiler and to specify the language on the command line with the -x option.

Reply:
I reverted this change and gcc-7 and gcc-8 seem to work OK. No point in forcing g++.